downloader.go

```diff
@@ -303,7 +307,12 @@ func (d *Downloader) Download(urls ...string) <-chan DownloadStatus {
 // NewDownloader creates a downloader with the defalt configuration. Check
 // the constants in this package for their values.
 func DefaultDownloader() *Downloader {
+	dir, err := os.Getwd()
+	if err != nil {
+		dir = ""
```

Comment:
Shouldn't we stay with the default temporary or downloads folder from the OS?

Reply:
What I have in mind is the CLI tools usually used to automate downloads, such as `curl` or `wget`: they default to the current working directory.

Why not OS's temporary directory? In case the user is distracted, they might just lose the work of hours with a simple restart (really common in the container's world).

Why not download folder? This is a concept of GUI systems and is not applicable to all OS and usages (if I'm not wrong, for example, creating a user via CLI in Debian does not create a download folder).

Reply:
Makes total sense! Closing this comment